lib/folio_client/token_wrapper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require "byebug"
+class FolioClient
+  # Wraps API operations to request new access token if expired
+  class TokenWrapper
+    def self.refresh(config, connection, &block)
```

Comment:
I think you should be able to remove the explicit block arg here. (Missed that in the GlobusClient PR.)

Reply:
cool, removed

Reply:
I just removed in globus_client with a new commit on the same PR you just approved.

Reply:
🍻